maven-scm-providers/maven-scm-provider-hg/pom.xml

```diff
@@ -66,10 +66,17 @@
     </plugins>
   </build>
 
-
   <profiles>
     <profile>
       <id>tck-hg</id>
+      <activation>
+        <property>
+          <name>run-its</name>
+        </property>
+        <file>
+          <exists>/usr/bin/hg</exists>
```

Comment:
Not portable

Reply:
I agree. I'll refactor using the assumeTrue you mentioned.